README.md

```diff
@@ -71,23 +67,20 @@ jobs:
    runs-on: [windows-2019]
    env:
       server: 'iis-server.domain.com'
```

Comment:
Since it didn't change here I couldn't check what the job name was.  In a few of the other repos the job name needed to be updated, so you may want to check that it reflects what's going on here too.